### ISSUE TITLE
Change copypastor_url to copypastor_host

### ISSUE THREAD

**Issue by FelixSFD** [SOBotics/Guttenberg#194]
The RegEx in the `feedback`-command contains the URL entered in `copypastor_url` in the login properties. This can lead to problems. If it's set to HTTPS, the command won't accept HTTP-links.

By changing the property to `copypastor_host`, we can easily modify the RegEx to accept both

**Comment by juliusfriedman** [comment]
> The RegEx in the `feedback`-command contains the URL entered in `copypastor_url` in the login properties. This can lead to problems. If it's set to HTTPS, the command won't accept HTTP-links.
> 
> By changing the property to `copypastor_host`, we can easily modify the RegEx to accept both

I agree on fixing bugs so long as they are documented to not cause changes in behavior which are then therefor hard to define or explain in context to others (bugs etc.)

Please let me know where I can find the actual license and disclaimer amongst the other information requried to contribute to making som botz.

Especially flying 1x

**Comment by juliusfriedman** [comment]
See also Sed or Awk

**Comment by juliusfriedman** [comment]
And I don't like cats n e more than U do :paintbrush:

**Comment by Bhargav-Rao** [comment]
Hi, thank you for helping out. We have some guidelines for contributing here https://github.com/SOBotics/Guttenberg/blob/develop/CONTRIBUTING.md. You can always ask us in chat or in here if you want any clarification. Thanks.
